### PR TITLE
draft: upgrading from nRF5 SDK to NCS

### PR DESCRIPTION
**Pull request by gminn**
### Summary

Draft for Ultrahuman's article on nRF5 SDK -> NCS journey.

Notable adjustments in this commit:
- Uncomment commented code
- Convert caps sub-headings to markdown sub-headings
- Convert bash script image to code snippet

### Test Plan

Visual check

### CODE REVIEW

**Comment by gminn** [comment]
* **#537** <a href="https://app.graphite.dev/github/pr/memfault/interrupt/537?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/memfault/interrupt/537?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `master`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: _drafts/upgrading-from-nrf5-sdk-to-ncs.md [hunk 1]
```diff
@@ -127,7 +123,9 @@ static bool softdevice_info_ok(uint32_t sd_start_addr, uint32_t sd_size)
     if (SD_MAGIC_NUMBER_GET(sd_start_addr) != SD_MAGIC_NUMBER)
     {
         NRF_LOG_ERROR("The SoftDevice does not contain the magic number identifying it as a SoftDevice.");
-        //result = false;
+
+        // Protection removed for bootloader migration
```
Comment (gminn) [inline _drafts/upgrading-from-nrf5-sdk-to-ncs.md:127]: Added a comment for clarity

File: _drafts/upgrading-from-nrf5-sdk-to-ncs.md [hunk 2]
```diff
@@ -163,7 +162,9 @@ static uint32_t sd_activate(void)
     if (SD_MAGIC_NUMBER_GET(src_addr) != SD_MAGIC_NUMBER)
     {
         NRF_LOG_ERROR("Source address does not contain a valid SoftDevice.")
-        //return NRF_ERROR_INTERNAL;
+        
+        // Protection removed for bootloader migration
```
Comment (gminn) [inline _drafts/upgrading-from-nrf5-sdk-to-ncs.md:166]: Same comment here as above

Comment (gaurav-2912) [reply _drafts/upgrading-from-nrf5-sdk-to-ncs.md:166]: Sure the changes look good

**Review by gaurav-2912 — APPROVED** [review]

**Review by noahp — APPROVED** [review]

**Pull request merged**